lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -6702,7 +6710,42 @@ namespace Js
         Assert(element1 != NULL);
         Assert(element2 != NULL);
 
+#ifndef ENABLE_DEBUG_CONFIG_OPTIONS
```

Comment:
This is not needed, since your flag has a default value of `false` the compiler will remove the else branch

Reply:
I've change to use CONFIG_FLAG such that this holds, and have removed the #ifndefs.